src/utils.ts

```diff
@@ -244,3 +244,30 @@ export async function createPhysicalDocument(
     const document: vscode.TextDocument = await vscode.workspace.openTextDocument(filePath);
     await vscode.window.showTextDocument(document, { preview: false });
 }
+
+/**
+ * Create a dummy dataset.
+ * @return A dummy dataset.
+ */
+export function createDummyDataset(): Dataset {
```

Comment:
Move this function to `__tests__` folder please. It's not related to main functionality.

Reply:
@shciubrcom seems that by default jest search for unit test everything is under __test__ folder, so in order to keep there the util service for dummy dataset I added an ignore pattern in the jest configuration, smth like this: "<rootDir>/src/__tests__/utils", is it ok for u or do u know other better way to do that?

Reply:
I think it's ok